### ISSUE TITLE
[Screen reader -Howdy-for-pms]: Lang attribute is not defined for the Howdy for pms page.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#14]
**User Experience**
Users who rely on screen readers will confuse to navigate the page if Lang attribute is defined incorrectly.

**Test Environment**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://hcl-szz1925.howdy.ai/dashboard
Browser: Edge
 
**Repro Steps**
Step 1: Open the URL: https://hcl-szz1925.howdy.ai/dashboard
Step 2: Navigate to "Dashboard" tab and select it.
Step 3: Navigate to "Howdy for Your PM" link and select it.
Step 4: "Howdy for Your PM" page gets opened.
Step 5: Open the developer tool bar by using F12 key
Step 6: Observe the issue.

**Actual Result**
Lang attribute is not defined for the Howdy for pms page. 

**Expected Result**
  Lang attribute should be defined Howdy for pms page.
Ex: lang = “en-us”.

**Suggested Fix**
The <html> element does not have a lang attribute

**Mas Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={eb8515c6-18d7-4af0-b47e-c46958b1b50e}
![MAS3 1 1_Language-HOWDY PM](https://user-images.githubusercontent.com/48272005/54473488-78796e00-47fe-11e9-9080-2273e2de7418.png)
![MAS3 1 1_Lang](https://user-images.githubusercontent.com/48272005/54473492-8202d600-47fe-11e9-9f71-310b678ae8d3.png)

**Comment by benbrown** [comment]
This page has been deprecated and removed from the site.

**Comment by v-gasikk** [comment]
Now this page is not available. Closing this issue based on developer comment.
